buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/plugins/performance/PerformanceTestPlugin.kt

```diff
@@ -103,6 +105,35 @@ class PerformanceTestPlugin : Plugin<Project> {
         tasks.register("performanceReport")
     }
 
+    private
+    fun Project.registerForkPointDistributionTask() {
+        val determineForkPoint = tasks.register("determineForkPoint", DetermineForkPoint::class)
+        val buildForkPointDistribution = tasks.register("buildForkPointDistribution", BuildForkPointDistribution::class) {
+            dependsOn(determineForkPoint)
+        }
+
+        buildForkPointDistribution.get().forkPointDistributionVersion.set(determineForkPoint.get().forkPointDistributionVersion)
+
+        tasks.register("configurePerformanceTestBaseline") {
+            dependsOn(buildForkPointDistribution)
+            doLast {
+                val commitBaseline = determineForkPoint.get().forkPointDistributionVersion.get()
+                project.tasks.withType(DistributedPerformanceTest::class) {
```

Comment:
Why do we only configure `DistributedPerformanceTest`? Shouldn't that feature be also available for ad-hoc performance tests? Or are we going to do this later?

Reply:
I think I want to make it step by step. For now let's only enable it for `DistribnutedPerformanceTest`.